File: typescript/iniciante/1005.ts
import { bold, cyan } from "https://deno.land/std@0.224.0/fmt/colors.ts";

const A = Number(prompt("Digite a primeira nota (A):")?.replace(",", "."));
const B = Number(prompt("Digite a segunda nota (B):")?.replace(",", "."));

const weightA = 3.5;
const weightB = 7.5;

const average = (A * weightA + B * weightB) / (weightA + weightB);

console.log(cyan(`MEDIA = ${bold(average.toFixed(5))}`));

File: swift/iniciante/1005.swift
import Foundation

if let inputA = readLine(), let inputB = readLine(),
   let A = Double(inputA.replacingOccurrences(of: ",", with: ".")),
   let B = Double(inputB.replacingOccurrences(of: ",", with: "."))
{
    let weightA = 3.5
    let weightB = 7.5

    let average = (A * weightA + B * weightB) / (weightA + weightB)

    print(String(format: "MEDIA = %.5f", average))
}
